### ISSUE TITLE
out of date documentation

### ISSUE THREAD

**Issue by dlyr** [STORM-IRIT/Radium-Engine#639]
Documentation refers to snippets in `HelloRadium/minimalradium.cpp` which has been removed (commit 6df619d1b0dd49b069842b92f016494183d8c7df)
Some snippets are defined, but current doc on material is broken.
Please update ;)

**Comment by dlyr** [comment]
PR #698 do not closes this one, @MathiasPaulin doc need update.

**Comment by MathiasPaulin** [comment]
> PR #698 do not closes this one, @MathiasPaulin doc need update.

I agree. But writing doc only to write doc is not so motivating ... I'll do that little to little.
